docs/resources.md

```diff
@@ -181,10 +181,10 @@ spec:
     resources:
     - name: builtImage
       type: image
-      outputImagePath: /workspace/builtImage
+      outputImageDir: /workspace/builtImage
```

Comment:
`builtImage` diverges from `buildImage` in the text on line 169.

Reply:
fixed